### PR TITLE
Refine numbers in headers of summary table

### LINKED ISSUE #15: total number in the header should not be just a sum
ATM it has `Passed (295/34452) | Failed (103/17930) | Timed Out (66/2626)` in which each number is just a sum across two tests (pynwb_open_load_ns and matnwb_nwbRead) rows.  It is not interesting to see in the header just the sum -- we would want to see for "Passed" to be a `len(intersection(succeeded))` and for "Failed" or "Timed out" `len(union(failed))` items (dandisets or individual files), so we could see how many good (pass all tests - could be opened with **any** of the tool) dandisets/files  we have, and how many dandisets/files have issue (would fail with **some** tool).

### PR DESCRIPTION
**Pull request by jwodder**
Closes #15.

### CODE REVIEW

File: healthstatus.py [hunk 1]
```diff
@@ -191,17 +191,14 @@ async def dowork(rec: MemoryObjectReceiveStream[Dandiset]) -> None:
             }
             test_summaries = {tn: TestSummary(tn) for tn in TEST_NAMES}
             for r in all_reports:
+                passed, failed, timedout = r.combined_counts()
+                asset_qtys[Outcome.PASS] += passed
+                asset_qtys[Outcome.FAIL] += failed
+                asset_qtys[Outcome.TIMEOUT] += timedout
+                dandiset_qtys[Outcome.PASS] += bool(passed)
```
Comment (yarikoptic) [inline healthstatus.py:198]: I think this is wrong logic which would report dandiset if any asset passed any test, isn't it so?  don't we need smth like

```suggestion
                dandiset_qtys[Outcome.PASS] += bool(not failed and not timedout)
```

?

Comment (jwodder) [reply healthstatus.py]: Right, I missed that you said the number of passing Dandisets should be "where all assets pass all tests".

**Comment by yarikoptic** [comment]
cool. Could you actually rerender that header stats so we would see the results -- all the information is there, right?

**Comment by jwodder** [comment]
@yarikoptic The information is there in the `status.yaml` files, but the script isn't built for getting information back out of them.  Unless you foresee the README repeatedly needing to be regenerated from the `status.yaml` files, I don't think updating script to do this is worth it.

**Comment by yarikoptic** [comment]
ok, it would be a separate development then anyways. Given that it takes days (or weeks) for a full pass ATM I am afraid we aren't to see the effect of this fix for awhile.

**Pull request merged**